### DISCUSSION TITLE
Does GO no longer support mono samples?

### DISCUSSION THREAD

**Discussion by vpoguru** [GrandOrgue/grandorgue#1593] [Help]
When I attempt to load mono samples in GO, I encounter the error message: Invalid WAV file.

The help file states the following:

The samples are stored as WAV files according to the WAV file specification. The supported formats are: 8 bit, 16 bit and 24 bit PCM or 32 bit IEEE float, either mono or stereo.

Is there a setting that needs to be changed?

**Comment by larspalo** [comment]
There must be something else that's wrong with your samples (like a malformed wav header or maybe wrong order of chunks). It's still perfectly possible to load mono samples.
